### ISSUE TITLE
upgrade to 1.4 is missing a variable definition

### ISSUE THREAD

**Issue by enoodle** [openshift/openshift-ansible#2926]
When running ansible-playbook -i 
```./inventory_file playbooks/byo/openshift-cluster/upgrades/v3_4/upgrade.yml```
I encountered the error:
```
TASK [Create the master service env file] **************************************
fatal: [vm-48-13.eng.lab.tlv.redhat.com]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'openshift_master_config_file' is undefined"}
```

I noticed that the file
```
playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml
```

Is including `../../../../roles/openshift_master/vars/main.yml` which uses `roles/openshift_master/templates/atomic-openshift-master.j2` which needs "openshift_master_config_file" but it is not defined in `../../../../roles/openshift_master/vars/main.yml`. Adding the file `../../../../roles/openshift_master_facts/vars/main.yml` in this task list solves it. I would have opened a PR but I have a feeling that this isn't the desired solution here.

cc @detiber @dgoodwin 

[1]https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml#L70

**Comment by dgoodwin** [comment]
Thanks for reporting @enoodle , this should be fixed now via #2913 and https://bugzilla.redhat.com/show_bug.cgi?id=1400819

**Comment by enoodle** [comment]
:+1:
